exporter/awsxrayexporter/internal/translator/aws_test.go

```diff
@@ -182,25 +182,7 @@ func TestAwsFromEksResource(t *testing.T) {
 }
 
 func TestAwsWithAwsSqsResources(t *testing.T) {
-	instanceID := "i-00f7c0bcb26da2a99"
-	containerName := "signup_aggregator-x82ufje83"
-	containerID := "0123456789A"
 	resource := pcommon.NewResource()
-	attrs := pcommon.NewMap()
-	attrs.PutStr(conventions.AttributeCloudProvider, conventions.AttributeCloudProviderAWS)
-	attrs.PutStr(conventions.AttributeCloudAccountID, "123456789")
-	attrs.PutStr(conventions.AttributeCloudAvailabilityZone, "us-east-1c")
-	attrs.PutStr(conventions.AttributeContainerName, containerName)
-	attrs.PutStr(conventions.AttributeContainerImageName, "otel/signupaggregator")
-	attrs.PutStr(conventions.AttributeContainerImageTag, "v1")
-	attrs.PutStr(conventions.AttributeK8SClusterName, "production")
-	attrs.PutStr(conventions.AttributeK8SNamespaceName, "default")
-	attrs.PutStr(conventions.AttributeK8SDeploymentName, "signup_aggregator")
-	attrs.PutStr(conventions.AttributeK8SPodName, "my-deployment-65dcf7d447-ddjnl")
-	attrs.PutStr(conventions.AttributeContainerName, containerName)
-	attrs.PutStr(conventions.AttributeContainerID, containerID)
-	attrs.PutStr(conventions.AttributeHostID, instanceID)
-	attrs.PutStr(conventions.AttributeHostType, "m5.xlarge")
```

Comment:
The following lines were removed as they were not actually doing anything, nor were they being asserted as part of the test.

Reply:
For some of the other tests the created `attrs` are attached to the `resource`. However, here this isn't happening. The `attrs` variable is scoped to the test function and is never associated with `resource` and thus never gets passed to the function being tested. It actually does nothing 😅.

If you still think I should revert this, I'll happily do so.

Reply:
@Aneurysm9 does this still need addressing?

Reply:
Yes.  This is unrelated to the change under test and should be done separately if required.

Reply:
Reinstated.